modules/ROOT/pages/prereq-hardware.adoc

```diff
@@ -90,52 +97,24 @@ sudo systemctl start var-lib-gravity-planet-etcd.mount
 
 == Docker Device
 
-Unless specified, Docker configuration defaults to the use of Device Mapper in loopback mode (using `/dev/loopX` devices), which is not recommended for production. To configure Docker to use a dedicated device for Device Mapper storage driver, an unformatted device or a partition (/dev/sdd) can be provided during installation. This directory is automatically configured and set up for use.
+Unless specified, Docker configuration defaults to the use of the overlay driver, which is recommended for production.
 
-Unformatted devices potentially usable for system directory / Device Mapper are automatically discovered by agents running on each node. Discovered devices are offered on a drop-down menu for configuration before the installation is started.
+Format this device as `xfs` or `ext4` and mounted as `/var/lib/gravity/planet/docker`.
 
-You can list unmounted devices with the following command:
-----
-lsblk --output=NAME,TYPE,SIZE,FSTYPE -P -I 8,9,202|grep 'FSTYPE=""'
-----
+The following example shell snippet mounts the Docker device using systemd mount files to the path:
+`/var/lib/gravity/planet/docker`. You can also mount the Docker device using `/etc/fstab` or another method.
+Consult with your system administrator to determine the best way to meet this requirement.
+If you use the following example as a guide for this process, specify the correct device name in two places.
 
-Unmounted devices have an empty value in FSTYPE column. Devices with TYPE="part" are partitions on another device. The following command lists only specific device types:
-
-|===
-|Device type|Description
-|8   |SCSI disk devices
-|9   |Metadisk (RAID) devices
-|202 |Xen virtual block devices (Amazon EC2)
-|===
-
-=== Reset Docker Device
-
-Using Logical Volume Manager, you can group multiple physical volumes into a single storage volume (volume group) and then divide these into logical volumes. Physical volumes are either a whole device or a partition.
-
-When a Docker device is in use by another logical volume or you want to manually reset a Docker device previously configured for Device Mapper, perform the following commands to reset the Docker device:
+[NOTE]
+Mount this device AFTER you mount the System State device under `/var/lib/gravity/planet`. If you are using system.d units to mount the volumes, declare mount dependencies using `After=<device-name>.<mount>`
```

Comment:
Should "system.d" be "systemd", which is what you have other text?

Reply:
Yes - good catch!